packages/@okta/vuepress-site/docs/reference/architecture-center/multiple-environments/lab/index.md

```diff
@@ -22,57 +22,31 @@ The tutorials in this lab are designed to run sequentially, with each tutorial s
 
 ## Prerequisites
 
-You'll need to set up the following to complete the tutorials in this lab.
+You need to set up the following to complete the tutorials in this lab.
 
-## Applications
+## Two Okta accounts
 
-This lab uses [Okta CLI](https://cli.okta.com/) to create Okta developer accounts quickly.
+This lab requires two Okta accounts, one to represent a development environment, and one to act as a production environment. Both accounts must use **Okta Identity Engine**. The production environment must have [Okta Workflows](https://www.okta.com/platform/workflows/) enabled.
 
-> **Tip:** Before running Okta CLI, consider adding the directory path to its executable to your `PATH` environment variable.
+1. Go to <https://developer.okta.com/signup/> and click **Sign up free for Developer Edition**.
+1. Fill in the form and click **Sign up** to create an Okta developer account. This represents your development environment.
+1. Go to <https://www.okta.com/free-trial/workforce-identity/> .
+1. Fill in the form and click **Get Started** to create a 30-day free trial account. This represents your production environment.
 
-## Two Okta developer accounts
+> **Note:** Okta developer accounts don’t have Okta Workflows enabled so you need to use the free trial account for the production environment.
 
-This lab requires two Okta developer accounts, one to represent a development environment, and one to act as a production environment. Both accounts must use **Okta Identity Engine**.
+> **Tip**: You need your Okta org domain to follow this tutorial. It looks like `dev-123456.okta.com`. See [Find your Okta domain](/docs/guides/find-your-domain/). Where you see `${yourOktaDomain}` in this guide, replace it with your Okta domain.
 
-> **Tip:** If you're using existing accounts and want to check the org is running Okta Identity Engine rather than Okta Classic Engine, check the footer on any page of the Admin Console for that org. The version number is appended with **E** for Identity Engine orgs and **C** for Classic Engine orgs.
+### Can you use existing accounts?
 
-Okta CLI is the quickest way to create an Okta org, so we recommend using it to create both new orgs. Alternatively, you can manually sign up instead.
+If you want to use existing Okta accounts instead of new ones, check that:
 
-1. Open your terminal.
-2. Run `okta register`, and enter your First name, Last name, Email address, and Country.
-3. Click or tap **Activate** in the account activation email that is sent to the email address that you provided.
+* Both orgs are running Okta Identity Engine rather than Classic Engine.
+* The _production_ org has Okta Workflows enabled.
```

Comment:
```suggestion
* The production org has Okta Workflows enabled.
```

Reply:
Again, done, but why de-emphasize the word?

Reply:
Our standard is not to use italics: https://oktawiki.atlassian.net/wiki/spaces/DOC/pages/722929640/InfoDev+Style+Guide#2.-Terminology-quick-reference   row 36